### PR TITLE
[Logging] Mention `#[AsMonologProcessor]`

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fixes https://github.com/symfony/symfony-docs/issues/16818

### CODE REVIEW

File: logging/processors.rst [hunk 1]
```diff
@@ -164,6 +164,30 @@ If you use several handlers, you can also register a processor at the
 handler level or at the channel level instead of registering it globally
 (see the following sections).
 
+When registering a new processor, instead of adding the tag manually in your
+configuration files, you can take full advantage of the ``#[AsMonologProcessor]``
```
Comment (94noni) [inline logging/processors.rst:168]: as you target v5.4 and this can run on php v7, perhaps need to give hint that this only works with php v8+

**Review by 94noni — APPROVED** [review]

File: logging/processors.rst (hunk 1 above)
Comment (alexandre-daubois) [reply logging/processors.rst:168]: I would say yes, but it seems that all `versionadded` of the doc about attributes are not displaying this information, even on 5.4 🤔

Comment (OskarStark) [reply logging/processors.rst:168]: Indeed, we avoid the explicit PHP hint for PHP 8, but thanks for raising it 👌🏻

File: logging/processors.rst [hunk 2]
```diff
@@ -164,6 +164,30 @@ If you use several handlers, you can also register a processor at the
...
+attribute by using it one the processor class::
```
Comment (OskarStark) [inline logging/processors.rst:169]:
```suggestion
attribute by using it on the processor class::
```

File: logging/processors.rst [hunk 3]
```diff
@@ -164,6 +164,30 @@ If you use several handlers, you can also register a processor at the
...
+
+    // src/Logger/SessionRequestProcessor.php
+    namespace App\Logger;
+
+    use Monolog\Attribute\AsMonologProcessor;
+    // ...
```
Comment (OskarStark) [inline logging/processors.rst:175]:
```suggestion
```

File: logging/processors.rst [hunk 4]
```diff
@@ -164,6 +164,30 @@ If you use several handlers, you can also register a processor at the
...
+    }
+
+The ``#[AsMonologProcessor]`` attribute takes 3 optional arguments:
+
+* ``channel``: the logging channel the processor should be pushed to
+* ``handler``: the handler the processor should be pushed to
+* ``method``: the method that processes the records (if the attribute is used
+  at the class level). Indeed, the attribute can also be used directly
+  on a class method.
+
```
Comment (OskarStark) [inline logging/processors.rst:190]: Versionadded missing

**Comment by javiereguiluz** [comment]
Thank you Alex!

Oskar, I did all the suggested changes while merging.

**Pull request merged**